pkg/network/ebpf/c/protocol-classification-defs.h

```diff
@@ -0,0 +1,33 @@
+#ifndef __PROTOCOL_CLASSIFICATION_DEFS_H
+#define __PROTOCOL_CLASSIFICATION_DEFS_H
+
+#include <linux/types.h>
+
+// Represents the max buffer size required to classify protocols .
+// We need to round it to be multiplication of 16 since we are reading blocks of 16 bytes in read_into_buffer_skb_all_kernels.
+// ATM, it is HTTP2_MARKER_SIZE + 8 bytes for padding,
+#define CLASSIFICATION_MAX_BUFFER (HTTP2_MARKER_SIZE + 8)
```

Comment:
`__declspec` is not supported on our compiler
furthermore, it reserves the number of bits, rather than multiplication that is required here
and that's a define, and it won't compile correctly in the code